wire-library/dependencies.gradle

```diff
@@ -24,14 +24,15 @@ ext.versions = [
 
 ext.deps = [
   plugins: [
+    android: 'com.android.tools.build:gradle:3.6.0',
+    androidDependency: 'com.android.tools.build:gradle:3.4.1',
```

Comment:
I don’t understand these two... comment?

Reply:
you should be able to `compileOnly` on `plugins.android` and not need `androidDependency`, if thats not the case per jesses comment would be good to have a comment here describing why

Reply:
I remove the redundancy. Since we are compile only we do whatever we want without forcing deps on consumers (no Android deps on non-android project).
I also updated to `4.0.1`